### PR TITLE
DPN-38/ make dropdown for units 

### PR DESCRIPTION
**Pull request by jxm0428**
Add dropdown for parameter 'length', 'r_not', 'r_a', 'x' and 'force'.
Add a test file 'test_ui_unit_field':
  -testing the select field and its options are created.
  -testing the selected value changes correctly.

### CODE REVIEW

**Review by trevorliberty — APPROVED** [review]
LGTM, make sure to pull down master before merging

File: selenium_tests/test_ui_unit_field.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# To run these tests, it is required that you perform the following two actions first:
+#   1. Install selenium webdriver and add it to your path variable
+#       https://zwbetz.com/download-chromedriver-binary-and-add-to-your-path-for-automated-functional-testing/
+#   2. Start the server running locally at the URL "http://localhost:8000/"
```
Comment (jjesse55) [inline selenium_tests/test_ui_unit_field.py:4]: This information is now in the README, so it isn't necessary for it to be in every file any longer.

**Review by valdiston — APPROVED** [review]

File: selenium_tests/test_ui_unit_field.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+from selenium import webdriver
+from django.test import LiveServerTestCase, tag
+from selenium.webdriver.common.keys import Keys
+from selenium_tests.selenium_test_base import SeleniumTestBase
+
+
+URL = "http://localhost:8000/"
```
Comment (jjesse55) [inline selenium_tests/test_ui_unit_field.py:12]: Are you using this in your test anywhere or is it just here due to copy and paste?

File: selenium_tests/test_ui_unit_field.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    """ Test unit select field"""
+
+    def __init__(self, methods):
+        super().__init__(methods)
+
+    def test_select_tag_is_created(self):
+        """ Test every unit's select field and options are exist, and the selected value changed correctly """
+
+        units = ['mm', 'cm', 'm', 'inch', 'feet']
+        parameters = ['length','r_not','r_a','x']
```
Comment (jjesse55) [inline selenium_tests/test_ui_unit_field.py:25]: these values will now have to be "r0" and "ra" now that Nate's PR got merged in

**Review by jjesse55 — CHANGES_REQUESTED** [review]
These should all be very minor and easy to fix! Other than those few things, LGTM

File: selenium_tests/test_ui_unit_field.py (hunk 2 above)
Comment (jxm0428) [reply selenium_tests/test_ui_unit_field.py]: Yeah I just copied from selenium_test_base file. I use the self.driver in the test_ui_unit_field file, so I think I need to import the SeleniumTestBase is that right?

Comment (jjesse55) [reply selenium_tests/test_ui_unit_field.py]: Correct. And inherit from it as you are doing

**Review by jjesse55 — APPROVED** [review]

**Pull request merged**